model/operations.py

```diff
@@ -175,6 +176,24 @@ def knockout_undo(model, to_undo):
             model.reactions.get_by_id(r_id).knock_out()
     return removed
 
+def changebounds_undo(model, to_undo):
```

Comment:
since `knockouts_undo` and `changebounds_undo` do the same thing, it's enough to rename `knockouts_undo` and reuse it in both cases